components/identity-i18n-mgt/org.wso2.carbon.i18n.mgt/src/main/java/org/wso2/carbon/i18n/mgt/config/Config.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.i18n.mgt.config;
+
+import java.util.Properties;
+
+public interface Config {
```

Comment:
No comments added to either Class or method definitions. Please add some meaningful comments explaining the semantics of this particular interface and its methods.